File: csharpcore/StarterBot/Entities/GameDetails.cs
using System;
using System.Collections.Generic;
using StarterBot.Enums;

namespace StarterBot.Entities
{
    public class GameDetails
    {
        public int Round { get; set; }
        public int MapWidth { get; set; }
        public int MapHeight { get; set; }
        public Dictionary<BuildingType, BuildingStats> BuildingsStats { get; set; }
    }
}

File: java/src/main/java/za/co/entelect/challenge/entities/GameDetails.java
package za.co.entelect.challenge.entities;

import za.co.entelect.challenge.enums.BuildingType;

import java.util.HashMap;

public class GameDetails {
    public int round;
    public int mapWidth;
    public int mapHeight;
    public int roundIncomeEnergy;
    public HashMap<BuildingType, BuildingStats> buildingsStats = new HashMap<>();

}
